MAUI/NavigationDrawer/Navigation-Pane-Sides.md

```diff
@@ -0,0 +1,154 @@
+---
+layout: post
+title: Configuring the Drawer in .NET MAUI Navigation Drawer | Syncfusion
+description: Learn here all about configuring the drawer in different sides support in Syncfusion .NET MAUI Navigation Drawer (SfNavigationDrawer) control and more.
+platform: maui
+control: NavigationDrawer
+documentation: ug
+---
+# Configuring The Drawer In Different Sides in .NET MAUI Navigation Drawer
+
+The supplemental pane can be pulled in and out from all four sides. The `Position` property is used to change the side of the pane, and the options are,
+
+*	[`Left`]()
+
+*	[`Right`]()
+
+*	[`Top`]()
+
+*	[`Bottom`]()
+
+N> The default position is [`Left`]().
+
+## Left
+
+The navigation pane pulled in and out from the `Left` side. It can be set as shown below:
```

Comment:
The navigation pane pulled in and out from the `Left` side can be set as follows:

Reply:
Changed